server-phoenix/lib/helios_web/schema/resolvers/location.ex

```diff
@@ -12,8 +12,10 @@ defmodule HeliosWeb.Schema.Resolvers.Location do
     {:ok, Jason.decode!(LocationData.location(System.get_env("PRIMARY_CITY_NAME")))}
   end
 
-  def weather(_parent, _args, _info) do
-    {:ok, Jason.decode!(WeatherData.weather)}
+  def weather(parent, _args, _info) do
+    response = WeatherClient.forecast(parent)
+    IO.inspect(Jason.decode!(response)["daily"])
```

Comment:
Looks like a debug line snuck in.

Reply:
They're always camouflaging within the code  😡